ONSdigital/florence
# Feature/react collections release schedule

### What

Add the feature to schedule a new collection by a release.

### How to review

User should be able to:

- Click the 'schedule by release' button and get a list of the release to choose from.
- Search through releases.
- Have the selected release played back on the create collection form.
- Able to change to a different publish type after selecting a release.

### Who can review

@jondewijones
